### ISSUE TITLE
incomplete reviews in published or archived submissions are counted as active

### ISSUE THREAD

**Issue by jnugent** [pkp/pkp-lib#9466]
**Describe the bug**
This is a follow up to https://github.com/pkp/pkp-lib/issues/8189.  We've a client who noticed that when assigning a reviewer, the number of active reviews did not align with what was shown when you logged in as that reviewer and viewed the dashboard.  It turns out that this reviewer has a number of Overdue reviews in submissions that were since published or declined.  The `date_completed` field in the `review_assignments` table is still null and the declined flag is 0.  This query:

https://github.com/pkp/pkp-lib/commit/e010f2cee357a936462825553ec7c3498fd3757b

Does not consider the submission status.

Perhaps this is by design since the intention of the "active review count" might be to provide editors with a sense of the reviewer's output, but it's a bit confusing since the reviews in non-active submissions no longer appear in the reviewer's dashboard.

**What application are you using?**
OJS 3.3.0.14

![Screenshot at 2023-10-27 12-49-07](https://github.com/pkp/pkp-lib/assets/218719/7075a286-89fa-4cbd-9e79-930519904f5f)

**Comment by Vitaliy-1** [comment]
I don't think that at the time of 3.3 or 3.4 release we had a clear criteria for  _active_  reviews. We started discussion regarding this just recently and implemented for 3.5. In the new dashboard it's limited only to reviews in the review stage.

> when assigning a reviewer, the number of active reviews did not align with what was shown when you logged in as that reviewer and viewed the dashboard

Yes, I don't think those numbers were aimed to be aligned and I would limit those not only by the status. Personally, I'd like not to change this for stable branches but it's something to watch out and check for the 3.5 release.
